.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!-- This form is for bug reports and feature requests ONLY! 
+
+If you're looking for help check [Vault's Google Group](https://groups.google.com/forum/#!forum/vault-tool).
+-->
+
+**Type of report:**
+
+[ ] Bug
+[ ] Feature request
+
+**Environment:**
```

Comment:
Environment encompasses the version as well, so it would show up like so:

**Environment:**

* **Version:**
* **Platform:**
* **Subsystem:**